lib/screens/qr_screen.dart

```diff
@@ -0,0 +1,187 @@
+import 'dart:io';
+
+import 'package:cupertino_will_pop_scope/cupertino_will_pop_scope.dart';
```

Comment:
It is used in the main. It was supposed to be used there as well, but after debugging a lot I found out that IOS swipeback gesture works, meaning it will pop the navigation stack as intended, but onWillPop is not called for some reason. On forums I've seen other developers face this issue with IOS and even my previous job encountered this issue. I wanted to reset the clock like how I did for android device so that the timers wont overlap. Reaching out to devs in discord returned no luck so I left it as it is.